.github/actions/push-image-aws-azure-gcp/action.yml

```
@@ -55,14 +55,9 @@ runs:
 
     - name: Push image to Amazon ECR
       shell: bash
-      env:
-        IMAGE: ${{ inputs.imageName }}
-        TAG: ${{ inputs.imageTag }}
-        REGISTRY: ${{ steps.login-ecr.outputs.registry }}
-        REPOSITORY: ${{ inputs.ecrRepository }}
       run: |
-        docker tag $IMAGE:$TAG $REGISTRY/$REPOSITORY:$TAG
-        docker push $REGISTRY/$REPOSITORY:$TAG
+        docker tag ${{ inputs.imageName }}:${{ inputs.imageTag }} ${{ steps.login-ecr.outputs.registry }}/${{ inputs.ecrRepository }}:$TAG
```

```suggestion
        docker tag ${{ inputs.imageName }}:${{ inputs.imageTag }} ${{ steps.login-ecr.outputs.registry }}/${{ inputs.ecrRepository }}:${{ inputs.imageTag }}
```

Plus i ty dva dole 🙈

---

dik

---

https://github.com/keboola/keboola-as-code/pull/2179/commits/cfcb37b26d64bb208c10b388b0dd8405cda7329c